anti-analysis/anti-debugging/debugger-detection/check-for-debugger-via-api.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: check for debugger via API
+    namespace: anti-analysis/anti-debugging/debugger-detection
+    author: michael.hunhoff@fireeye.com
+    scope: function
+    mbc:
+      - Anti-Behavioral Analysis::Detect Debugger
+    references:
+      - https://github.com/LordNoteworthy/al-khaser/blob/master/al-khaser/AntiDebug/CheckRemoteDebuggerPresent.cpp
+    examples:
+      - al-khaser_x86.exe_:0x420000
+  features:
+    - or:
+      - api: kernel32.IsDebuggerPresent
```

Comment:
i think some exception handlers and other compiler-inserted code may add calls to `IsDebuggerPresent`, especially in debug builds. so, i think there's a chance for FP here (well, its a true match, but probably not what people are expecting). is this ok? should we remove `IsDebuggerPresent`? need more research?

thoughts, everyone?

Reply:
I think more research on the FP / unexpected rate match makes sense. In the meantime, I'd vote to remove it.

Reply:
Good point - I've removed `IsDebuggerPresent` for now.